libs/blocks/tabs/tabs.css

```diff
@@ -49,39 +64,59 @@
   padding: var(--spacing-xxl) 0;
 }
 
-.tabs.xxl-spacing .paddle {
+.tabs:not(.quiet).xxl-spacing .paddle {
+  top: var(--spacing-xxl);
+}
+
+.tabs.quiet.xxl-spacing .paddle {
   top: var(--spacing-xxl);
```

Comment:
Do we need the repetition, since the same declaration applies to both quiet and non-quiet tabs? Same question applies to all spacing tokens. If there is a need for repetition, let's at least consolidate the selectors, since they have the same declaration.